package.json

```diff
@@ -18,7 +18,7 @@
   "licenses": "MIT",
   "dependencies": {},
   "devDependencies": {
-    "basisjs": "basisjs/basisjs#1.5",
+    "basisjs": "basisjs/basisjs#v1.5.1",
```

Comment:
yes, sure. Because it downloads package from npm, not github. But I see that you install dependences from github for some reason and I don't want change behavior without knowledge why.